src/app/shared/card-detail-layout/card-detail-layout.component.html

```diff
@@ -1,38 +1,42 @@
 <div class="container-fluid" style="padding-top: 10px">
     <h2>{{header}}</h2>
-    <div class="d-flex justify-content-center" *ngIf="pagination && items.length !== 0">
-        <ngb-pagination
-            *ngIf="pagination.totalPages > 1"
-            [(page)]="pagination.currentPage"
-            [pageSize]="pagination.itemsPerPage"
-            (pageChange)="onPageChange($event)"
-            [rotate]="false" [ellipses]="false" [boundaryLinks]="true"
-            [collectionSize]="pagination.totalItems"></ngb-pagination>
-    </div>
-
-    <div class="row no-gutters">
-        <div class="col-auto" *ngFor="let item of items; trackBy:trackByIdentity; index as i">
-            <ng-container [ngTemplateOutlet]="itemTemplate" [ngTemplateOutletContext]="{ $implicit: item, idx: i }"></ng-container>
+    <div class="content-container" *ngIf="!isLoading && items.length !== 0">
```

Comment:
You need else: noData to show the ng-template #noData when the if statement is false.